app/assets/v2/js/grants/fund.js

```diff
@@ -199,6 +200,11 @@ $(document).ready(function() {
         let approvalSTR = realApproval.toLocaleString('fullwide', { useGrouping: false });
 
         web3.eth.getAccounts(function(err, accounts) {
+          if (data.recurring_or_not == 'recurring' && grant_admin_address === accounts[0]) {
```

Comment:
shouldn't we check the user instead? I believe the address for the owner can be set to any address. Or also another grant team member can still go and auto fund.

Reply:
@octavioamu self funding with a different address is okay - theres actually a limitation in the contract itself that causes an error when you self fund with the grant admin's address

Reply:
@octavioamu @danlipert is it okay the current validation?